components/gc-page-success/gc-page-success-new-fr.html

```diff
@@ -0,0 +1,107 @@
+---
+{
+	"altLangPage": "gc-page-success-new-en.html",
+	"altLangPrefix": "gc-page-success_this-var-is-broken-in-modern-gcweb",
+	"breadcrumb": [ "this-var-is-broken-in-modern-gcweb" ],
+	"breadcrumbs": [
+		{ "title": "Accueil GCWeb", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-fr.html" }
+	],
+	"css": ["https://use.fontawesome.com/releases/v5.8.1/css/all.css"],
+	"dateModified": "2021-12-06",
+	"description": "Widget de rétroaction pour Canada.ca",
+	"language": "fr",
+	"title": "Widget succès de page (NEW) - Exemple pratique",
+	"noReportProblem": "true"
+}
+---
+{% include alert-provisional.html siteroot="../../" %}
+
+<style>
+@media screen and (max-width: 767px) {
+	.wb-enable .text-center-xs-js {
+		text-align: center;
+	}
+}
+
+@media screen and (min-width: 768px) {
+	.d-flex {
+		display: flex;
+	}
+
+	.align-items-center {
+		align-items: center;
+	}
+}
+</style>
+
+<div class="wb-prettify all-pre hide"></div>
+
+<div class="row row-no-gutters mrgn-tp-xl">
+	<div class="col-sm-7 col-lg-6">
+		<figure class="gc-pg-hlpfl provisional well mrgn-bttm-0" aria-live="polite">
+			<figcaption class="wb-inv">Rétroaction de la page</figcaption>
+			<form id="gc-pg-hlpfl-frm" action="/gc/services/generateemail" method="post" autocomplete="off" class="provisional wb-postback" data-wb-postback="{&quot;success&quot;:&quot;.gc-pg-hlpfl-thnk&quot;,&quot;content&quot;:&quot;#gc-pg-hlpfl-frm&quot;}">
```

Comment:
```suggestion
			<form id="gc-pg-hlpfl-frm" action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" autocomplete="off" class="provisional wb-postback" data-wb-postback="{&quot;success&quot;:&quot;.gc-pg-hlpfl-thnk&quot;,&quot;content&quot;:&quot;#gc-pg-hlpfl-frm&quot;}">
```
The `/gc/services/generateemailis` specific to AEM implementation. For non AEM implementations, maybe the TBS link could do.